packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -36,7 +36,7 @@ class RenderParagraph extends RenderBox {
   // TODO(abarth): This logic should live in TextPainter and be shared with RenderEditableLine.
   void _layoutText(BoxConstraints constraints) {
     assert(constraints != null);
-    assert(constraints.debugAssertIsNormalized);
+    assert(constraints.debugAssertIsValid());
```

Comment:
I think it's ignored.  We really need to re-work the Paragraph API to make actual sense.